splunktaucclib/rest_handler/endpoint/validator.py

```diff
@@ -78,6 +79,8 @@ def put_msg(self, msg, high_priority=False):
             self._msgs.insert(0, msg)
         else:
             self._msgs.append(msg)
+        if len(self._msgs) > self._max_queue_size:
```

Comment:
@harshpatel-crest I see a potential issue here if `high_priority` is `False` and it is for most of the validators (except `Port`, `Email` and `Host`). Let's assume that we are not using them and that can be the case.

```
b = []
b.append(0)
b.append(1)
b.append(2)
b.append(3)
b.append(4)
b.append(5)
b[:3]
```
After this you will end up with `b = [0, 1, 2]` which means that most recent messages are lost.

Reply:
@arys-splunk Yes but I think ultimately the handler uses the `msg` property method which shows the first message in list so I'm not sure if we even need this list in that case.

Reply:
We may also consider using `collections.deque` instead of `list`.

Reply:
Also, the self._msg is supposed to be an internal variable only. It's not exposed via any method.

Reply:
@harshpatel-crest What do you think then about removing the list and storing only the message? Then we don't need even to think about the size of validation message queue.

Reply:
On the TA side, we have `put_msg` exposed which allows a second argument that is used in many TAs so that's why I thought this was the most non-breaking fix we can have which needs no changes on the TA side.